.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -16,6 +16,8 @@ body:
       options:
         - label: "I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to."
           required: true
+        - label: "I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed."
```

Comment:
"problem" doesn't quite fit here in the context of a feature request, any suggestions?

Reply:
They're trying to solve a problem, though. That's why we have "Describe any problem or limitation you come across while using the app which would be solved by this feature." below.

Reply:
Hmm, does that apply to my other review comment too?

Reply:
Okay then, go ahead and commit that suggestion.